src/Commands/ToggleAutoShifting.cpp

```diff
@@ -2,17 +2,20 @@
 #include "TankDrive.h"
 #include "../Subsystems/Chassis.h"
 
-ToggleAutoShifting::ToggleAutoShifting(bool autoShift)
-: mAutoShift(autoShift) {
+ToggleAutoShifting::ToggleAutoShifting()
+: mAutoShift(nullptr) {
```

Comment:
`mAutoShift` is type `bool`. `nullptr` is an inappropriate initializer for a `bool`.